x-pack/plugins/data_visualizer/public/application/common/components/results_links/results_links.tsx

```diff
@@ -34,7 +47,7 @@ interface Props {
   timeFieldName?: string;
   createDataView: boolean;
   showFilebeatFlyout(): void;
-  additionalLinks: ResultLink[];
+  asyncLinkCards?: AsyncLinkCards;
```

Comment:
IMO the original `additionalLinks` name is better here.
Possibly changed to `getAdditionalLinks`?

Reply:
Updated here https://github.com/elastic/kibana/pull/131342/commits/966ff9f91c5ec282e16201a0f76f2e248aed2a86